pkg/registry/pod/etcd/etcd_test.go

```diff
@@ -544,10 +507,10 @@ func TestEtcdCreateBinding(t *testing.T) {
 		},
 	}
 	for k, test := range testCases {
-		storage, bindingStorage, _, fakeClient := newStorage(t)
+		storage, bindingStorage, _, server := newStorage(t)
```

Comment:
I don't disagree.  Are you opposed to follow up PR on this one.  The changeset is so large I would prefer to follow up then conflate it.  I would call it test cleanup from XYZ PR.